plans/dht/test/find_peers.go

```diff
@@ -71,10 +75,36 @@ func FindPeers(runenv *runtime.RunEnv) error {
 
 		t := time.Now()
 
+		ectx, cancel := context.WithCancel(ctx)
+		ectx, events := routing.RegisterForQueryEvents(ctx)
+		log := queryLog.With("target", p.ID)
```

Comment:
Perhaps, I'd move this to a function so we could hook it up on all tests as I did on #420.

Reply:
I'm going to block this on deciding how we want to log for now.